### ISSUE TITLE
Status bar not responding to dark/light theme.

### ISSUE THREAD

**Issue by techninja025** [mozilla-mobile/firefox-ios#13298]
Upper status bar (clock/connectivity/battery) font color not responding to device/app light or dark theme correctly. The mode pictured is supposed to be light mode. Device pictured is an iPhone XR. Firefox 109.0 iOS 16.3.1

![7FF70DE5-EE27-4307-AD35-00DE30834B75](https://user-images.githubusercontent.com/95654573/220403388-5978abdf-687c-4ff8-9df8-8be03e7160d0.png)

![D44D9575-6335-4A49-9BC0-CCE59AC674ED](https://user-images.githubusercontent.com/95654573/220403378-99863a86-2438-4ec2-9c99-2f44936a1647.png)

**Comment by lmarceau** [comment]
Hello! This duplicates https://github.com/mozilla-mobile/firefox-ios/issues/13269 I believe

**Comment by techninja025** [comment]
You are right. I didn’t see this originally. From: lmarceauSent: Tuesday, February 21, 2023 11:17 AMTo: mozilla-mobile/firefox-iosCc: techninja025; AuthorSubject: Re: [mozilla-mobile/firefox-ios] Status bar not responding to dark/light theme. (Issue #13298) Hello! This duplicates #13269 I believe—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by lmarceau** [comment]
Ok good to know, I'll close this issue in favor of the other one. If you think of any other details to add let us know, we've been having difficulties to reproduce this particular problem on our devices as it happens randomly.
